src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/BaseFunctionalTest.java

```diff
@@ -50,44 +52,43 @@ public void setUp() throws Exception {
         this.isProxyEnabled = Boolean.valueOf(config.getString("proxyout.enabled"));
         this.fluxFuncTest = config.getBoolean("flux-func-test");
 
-        StorageCredentialsAccountAndKey storageCredentials =
-            new StorageCredentialsAccountAndKey(
+        leaseClientProvider =  blobClient -> new BlobLeaseClientBuilder().blobClient(blobClient).buildClient();
+
+        StorageSharedKeyCredential storageCredentials =
```

Comment:
we can simplify with just `var`. and though avoid that new line. just personally buggers if it's new line :)